From 2a988273b382dba98131ed2f4097c4f1e01a11cb Mon Sep 17 00:00:00 2001
From: Osamu Aoki <osamu@debian.org>
Date: Mon, 13 May 2013 00:30:17 +0900
Subject: [PATCH 1/2] space/tab

---
 doc/EXPERIMENTAL |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/doc/EXPERIMENTAL
+++ b/doc/EXPERIMENTAL
@@ -13,8 +13,8 @@
 Here the optional type values and their defaults may be:
 
     type    Architecture    Multi-arch
-    all     all		        foreign     (all is always foreign)
-    any     any		        foreign
+    all     all             foreign     (all is always foreign)
+    any     any             foreign
     any     any             same        (if package name starts with "lib")
     foreign any             foreign
     same    any             same
