Description: Make generated debian/rules to be 755
Author: Osamu Aoki <osamu@debian.org>
--- debmake-4.0.0.orig/debmake/gen.py
+++ debmake-4.0.0/debmake/gen.py
@@ -195,6 +195,7 @@ override_dh_auto_install:
     # write to file
     ###################################################################
     writefile(para, 'debian/rules', msg)
+    os.chmod('debian/rules', 0o755)
     return
 
 #######################################################################
