From: Osamu Aoki <osamu@debian.org>
Date: Sat, 25 Oct 2014 21:41:59 +0900
Subject: Update template for autogen.sh

---
 extra0override/autogen | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/extra0override/autogen b/extra0override/autogen
index 60b9a59..24f59f6 100644
--- a/extra0override/autogen
+++ b/extra0override/autogen
@@ -1,6 +1,6 @@
 ### Initialize source with autogen.sh found in the upstream
 override_dh_autoreconf:
-	dh_autoreconf -- autogen.sh
+	dh_autoreconf -- ./autogen.sh
 ### Alternatively
 ### support for -Wl,--as-needed to ltmain.sh with autoreconf -f -i
 ###override_dh_autoreconf:
